testData/solutions/web.json

```diff
@@ -602,7 +603,8 @@
                             }
                         }
                     },
-                    "pictogramsEnabled": "http://registry\\.gpii\\.net/common/pictogramsEnabled"
+                    "pictogramsEnabled": "http://registry\\.gpii\\.net/common/pictogramsEnabled",
```

Comment:
Hey Christophe,

I think it would make sense to add this "common term" to the NP set and acceptance test as well.. I think it's a perfectly reasonable term, and since we have it in the solution entry, it makes sense to have tested as well.